doc/DeveloperGuide.md

```diff
@@ -88,6 +88,32 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to find persons with specific tag
+2. AddressBook shows a list of persons with specific tag
+3. User requests to rename the specific tag
+4. AddressBook requests for confirmation
+5. AddressBook renames the specific tag and shows a list of persons with renamed tag <br>
```

Comment:
The Use Case seeks to model the interaction between the Actor and System. In such a way user -> system -> user -> system -> etc.
Can you spot the place to modify? =P

Reply:
Between Step 4 and Step 5, User should confirm the renaming of tag?

Reply:
That would indeed be a better flow of events right =) ?